functions/exercise2/script.js

```diff
@@ -7,4 +7,17 @@ function isPalindrome(phrase) {
     }
     var reversedNormalizedPhrase = normalizedPhrase.split("").reverse().join("");
     return reversedNormalizedPhrase === normalizedPhrase;
+}
+
+function showResult() {
+    var phrase = document.getElementById("phrase").value;
+    try {
+        var result = isPalindrome(phrase);
+        if (result) {result = "is a palindrome";}
+        else {result = "isn't a palindrome ";}
+        document.getElementById("result").value = `${phrase} ${result}`;
+    }
+    catch (exception) {
+        document.getElementById("result").value = exception;
```

Comment:
You refer to document.getElementById("result") in two places. You may call it once, remember it in some variable and then refer it in two places to not repeat yourself.